examples/glamorous-jest-react/README.md

```diff
@@ -0,0 +1,9 @@
+## Example for `glamorous` with `jest` and `enzyme`
```

Comment:
Could you rename this directory to simply `examples/with-jest`. I think this is going to be our standard way to recommend people use this library with `jest` in general :+1: